sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/global-namespace-with-jsdom-unit-testing/common.webpack.config.js

```diff
@@ -11,14 +11,20 @@ module.exports = {
         loader: "imports-loader?" + Object.keys(globalModules).map(function(modName) {
           return modName + "=" + globalModules[modName];
         }).join(',')
-      },
-      Object.keys(globalModules).map(function(modName) {
-        // Expose global modules
-        return {
-          test: require.resolve(modName),
-          loader: "expose-loader?" + globalModules[modName]
-        }
-      })
+      }
     ]
   }
-};
\ No newline at end of file
+}
+
+Object.keys(globalModules).forEach(function(modName) {
+  // Expose global modules
+  config.module.loaders.push(
+    {
+      test: require.resolve(modName),
+      loader: "expose-loader?" + globalModules[modName]
+    }
+  );
+});
+
+
+module.exports = config;
```

Comment:
It's no the same syntax in v2, V3 or v1:

https://github.com/webpack-contrib/expose-loader#usage

```javascript
// webpack v1 usage
module: {
  loaders: [
    { test: require.resolve("jquery"), loader: "expose-loader?$" }
  ]
}

// webpack v2 usage
module: {
  rules: [{
          test: require.resolve('jquery'),
          use: [{
              loader: 'expose-loader',
              options: '$'
          }]
      }]
}
```